### PR TITLE
Pub: 3.7.15,3.8.15,3.9.15,3.10.8, and more. [release]

### PR DESCRIPTION
**Pull request by JalexChen**
Respin due to CVE

### CODE REVIEW

**Comment by felicianotech** [comment]
Why is this a draft?

**Comment by JalexChen** [comment]
@felicianotech Draft because i stepped away for a bit. Ready now.

File: 3.10/browsers/Dockerfile [hunk 1]
```diff
@@ -11,9 +11,6 @@ RUN curl -sSL -o selenium-server-standalone-${SELENIUM_VER}.jar "https://seleniu
     rm selenium-server-standalone-${SELENIUM_VER}.jar
 
 RUN sudo apt-get update && \
-	sudo apt-get install --yes --no-install-recommends \
-		xvfb \
-	&& \
```
Comment (felicianotech) [inline 3.10/browsers/Dockerfile:16]: Needs updated shared tools.

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: 3.10/browsers/Dockerfile (hunk 1 above)
Comment (JalexChen) [reply 3.10/browsers/Dockerfile]: Fixing

**Review by felicianotech — APPROVED** [review]

**Pull request merged**